aptos-move/framework/supra-framework/sources/automation_registry.move

```diff
@@ -80,7 +83,7 @@ module supra_framework::automation_registry {
     }
 
     // todo : this function should call during initialzation, but since we already done genesis in that case who can access the function
-    fun initialize(supra_framework: &signer) {
+    public(friend) fun initialize(supra_framework: &signer) {
```

Comment:
Does not necessarily have to be `(friend)` as long as we check that `signer` is indeed `supra_framework`, that way we can call this from governance script.